### PR TITLE
DRYing PickerTop.

### PR DESCRIPTION
**Pull request by gcedo**
Extracted PickerTop shared code.

### CODE REVIEW

File: src/PickerTop.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import cx from 'classnames';
+
+export default React.createClass({
+  propTypes: {
+    fixed: PropTypes.bool,
+    handleClick: PropTypes.func,
+    nextDate: PropTypes.func,
+    previousDate: PropTypes.func,
+    title: PropTypes.string,
+    textClassNames: PropTypes.string,
```
Comment (FrancescoCioria) [inline src/PickerTop.js]: you're using `textClassNames` to style `title` which is a bit weird, I would change `title` in `value` and `textClassNames` in `valueClassName`

File: src/daypicker/DayPickerTop.js [hunk 2]
```diff
@@ -29,31 +30,30 @@ const DayPickerTop = React.createClass({
...
-          <div className='react-datepicker-button button-right' onClick={partial(this.changeMonth, (month + 1))}>
-            &gt;
-          </div>
-        </div>
-        <div className='week-days'>
-          {daysMin}
-        </div>
-      </div>
+      <PickerTop
+        fixed={fixed}
```
Comment (FrancescoCioria) [inline src/daypicker/DayPickerTop.js]: shouldn't this be a boolean? I think you're passing a string

File: src/monthpicker/MonthPickerTop.js [hunk 3]
```diff
@@ -25,19 +26,13 @@ const MonthPickerTop = React.createClass({
...
-          <div className={`react-datepicker-button button-label ${fixed}`} onClick={this.changeMode}>
-            <strong className={this.props.textClassNames}>{year}</strong>
-          </div>
-          <div className='react-datepicker-button button-right' onClick={partial(this.changeYear, (year + 1))}>
-            &gt;
-          </div>
-        </div>
-      </div>
+      <PickerTop
+        fixed={fixed}
```
Comment (FrancescoCioria) [inline src/monthpicker/MonthPickerTop.js]: same here

**Comment by FrancescoCioria** [comment]
Thank you for the PR, this is very useful to better organize the code!
I am not totally convinced by the API of `PickerTop` though, I made an inline comment.

File: src/PickerTop.js (hunk 1 above)
Comment (gcedo) [inline src/PickerTop.js]: I just copied the name from the existing file :blush: I'll change it right away.

**Comment by FrancescoCioria** [comment]
thanks :)

**Pull request merged**